src/model.hpp

```diff
@@ -89,14 +110,14 @@ class model {
     }
 
     // one cell per group:
-    model(const recipe& rec):
-        model(rec, util::partition_view(util::make_span(0, rec.num_cells()+1)))
+    model(const recipe& rec, backend_policy policy):
+        model(rec, util::partition_view(util::make_span(0, rec.num_cells()+1)), policy)
```

Comment:
I removed the default value from the other constructor: now both require that the policy is passed explicitly.